packages/example-remix-pages-app/tests/index.test.ts

```diff
@@ -26,23 +26,32 @@ describe("Remix", () => {
   beforeAll(async () => {
     spawnSync("npm", ["run", "build"], {
       shell: isWindows,
-      cwd: resolve(__dirname, "../"),
+      cwd: path.resolve(__dirname, "../"),
     });
     wranglerProcess = spawn("npm", ["run", "dev:wrangler"], {
       shell: isWindows,
-      cwd: resolve(__dirname, "../"),
+      cwd: path.resolve(__dirname, "../"),
       env: { BROWSER: "none", ...process.env },
     });
-    wranglerProcess.stdout.on("data", (chunk) => {
+    wranglerProcess.stdout?.on("data", (chunk) => {
       console.log(chunk.toString());
     });
-    wranglerProcess.stderr.on("data", (chunk) => {
+    wranglerProcess.stderr?.on("data", (chunk) => {
       console.log(chunk.toString());
     });
   });
 
-  afterAll(() => {
-    wranglerProcess.kill();
+  afterAll(async () => {
```

Comment:
could probably clean this up by using execa here, but I'll leave that for some other time/person